### PR TITLE
LinkInBio template with LaunchDarkly integration

### PR DESCRIPTION
**Pull request by erinmikailstaples**
Adds a link in bio template with a LaunchDarkly integration.

### CODE REVIEW

**Review by masenf — COMMENTED** [review]
can you please rebase with the `.venv` directory removed

**Comment by erinmikailstaples** [comment]
Can do! — one moment!

*Erin Mikail Staples* (she/her)
Dev Advocate | Community Human | Tech Education | Comedian

📧 ***@***.***
🌐 erinmikailstaples.com ( https://ed515793-d623-44e9-a1d2-ee17f996e35d.mailbutler.link/p2/e5a62510-8d86-4d96-aa30-8968280fefea/02d781a6-c621-4475-aee1-6d43c93ebcdb?contactid=e5ef6ec8-6a4e-3f76-496a-007cce585c54 )
☎️ let's chat ( https://cal.com/erinmikail )

*I’m very online — find me on…*
💼 LinkedIn ( https://ed515793-d623-44e9-a1d2-ee17f996e35d.mailbutler.link/p2/e5a62510-8d86-4d96-aa30-8968280fefea/c67554fa-344d-4de2-8121-70ba5820f2fc?contactid=e5ef6ec8-6a4e-3f76-496a-007cce585c54 ) | 🛠️ Polywork ( https://ed515793-d623-44e9-a1d2-ee17f996e35d.mailbutler.link/p2/e5a62510-8d86-4d96-aa30-8968280fefea/ad28b931-5368-48a6-9e26-411f187026e7?contactid=e5ef6ec8-6a4e-3f76-496a-007cce585c54 ) | 👩🏼‍💻 GitHub ( https://ed515793-d623-44e9-a1d2-ee17f996e35d.mailbutler.link/p2/e5a62510-8d86-4d96-aa30-8968280fefea/7703655c-9d10-4ece-81e2-8cf45437c59f?contactid=e5ef6ec8-6a4e-3f76-496a-007cce585c54 )
🔗 Mastodon ( https://ed515793-d623-44e9-a1d2-ee17f996e35d.mailbutler.link/p2/e5a62510-8d86-4d96-aa30-8968280fefea/993a3220-3276-40da-9f06-4866a4d5a8e9?contactid=e5ef6ec8-6a4e-3f76-496a-007cce585c54 ) | 📸 Instagram ( https://ed515793-d623-44e9-a1d2-ee17f996e35d.mailbutler.link/p2/e5a62510-8d86-4d96-aa30-8968280fefea/489f4812-bf31-4a35-aa78-43df684b8317?contactid=e5ef6ec8-6a4e-3f76-496a-007cce585c54 ) | 📆 Events ( https://ed515793-d623-44e9-a1d2-ee17f996e35d.mailbutler.link/p2/e5a62510-8d86-4d96-aa30-8968280fefea/502bf7d8-be50-4df9-850e-ed6335dd519b?contactid=e5ef6ec8-6a4e-3f76-496a-007cce585c54 )
🎥 YouTube ( https://ed515793-d623-44e9-a1d2-ee17f996e35d.mailbutler.link/p2/e5a62510-8d86-4d96-aa30-8968280fefea/e7c3e946-29d1-4501-a6a2-8c98643c71de?contactid=e5ef6ec8-6a4e-3f76-496a-007cce585c54 ) | 📹 Twitch ( https://ed515793-d623-44e9-a1d2-ee17f996e35d.mailbutler.link/p2/e5a62510-8d86-4d96-aa30-8968280fefea/24f75329-b262-4c78-9a1f-0eff0f8e569f?contactid=e5ef6ec8-6a4e-3f76-496a-007cce585c54 ) | 📧 Newsletter ( https://ed515793-d623-44e9-a1d2-ee17f996e35d.mailbutler.link/p2/e5a62510-8d86-4d96-aa30-8968280fefea/a62e540e-eec1-4158-8607-26f783efea9e?contactid=e5ef6ec8-6a4e-3f76-496a-007cce585c54 )

👋🏻 my working hours may not be your working hours, please don’t feel obliged to respond immediately

**Comment by erinmikailstaples** [comment]
@masenf removed the .venv directory — branch updated. 

erinmikailstaples:erin-final-edits

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "main"
```
Comment (elviskahoro) [inline .vscode/settings.json:3]: Could we ignore this file?

**Comment by elviskahoro** [comment]
@erinmikailstaples I'm running the tests! Will message you when they're done!

**Comment by erinmikailstaples** [comment]
hey @elviskahoro anything i can do to help with these tests?

looking at it looks like its failing b/c its not picking up the ldclient import — going to check the requirements file.

File: .vscode/settings.json (hunk 1 above)
Comment (elviskahoro) [reply .vscode/settings.json:3]: or delete it even?

**Review by elviskahoro — APPROVED** [review]

**Pull request merged**